<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VentanaBandeja</class>
 <widget class="QWidget" name="VentanaBandeja">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>313</width>
    <height>175</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>313</width>
    <height>175</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>313</width>
    <height>175</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Bandeja</string>
  </property>
  <property name="windowIcon">
   <iconset resource="imagenes.qrc">
    <normaloff>:/imagenes/Imagenes/iconoHorno.png</normaloff>:/imagenes/Imagenes/iconoHorno.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="tiempo">
         <property name="text">
          <string>Tiempo (s)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="capacidad">
         <property name="text">
          <string>Capacidad</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="cargadas">
         <property name="text">
          <string>Cargadas</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="horneadas">
         <property name="text">
          <string>Horneadas</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>6</number>
       </property>
       <item>
        <widget class="QLineEdit" name="editorTiempo">
         <property name="enabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="editorCapacidad">
         <property name="enabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="cargadasValor">
         <property name="text">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="horneadasValor">
         <property name="text">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="configurar">
     <property name="text">
      <string>Configurar</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="aceptar">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Aceptar</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="imagenes.qrc"/>
 </resources>
 <connections/>
</ui>
